lib/client/ipam.go

```diff
@@ -589,22 +655,17 @@ func (c ipams) RemoveIPAMHost(host *string) error {
 	// Determine the hostname to use.
 	hostname := decideHostname(host)
 
-	// Release host affinities.
```

Comment:
Well, I could leave it.  However, removing the IPAM host actually removes all the affinities anyway so I took it out.

I'm not a super big fan of it, but I suppose it's sort of like removing a tier and thus it removes all the policies within the tier with it.